importer.php

```diff
@@ -1,7 +1,9 @@
 <?php
+
 namespace TheRealMVP;
-use TheRealMVP\Importer\DBImport;
-use TheRealMVP\Importer\GetAPI;
+
+use TheRealMVP\Importers\DBImport;
```

Comment:
Not sure how it happened, but this file seems to be broken. It's missing two things:

1) requiring the autoloader
2) Creating a PDO connection to pass to DBImport